### PR TITLE
Compute Engine - Documentation 

### PR DESCRIPTION
**Pull request by themiszamani**
use of tables instead of lists, markdown changes, typos

### CODE REVIEW

File: doc/compute-input.md [hunk 1]
```diff
@@ -257,10 +257,10 @@ The full avro specification of the weight file:
 Downtime information: the period (start_time --> end_time) in which a specific service endpoint was in scheduled downtime. This information resided in the corresponding downtime avro file. The file has the following fields:
 
 |Name|Description|Required|
```
Comment (skanct) [inline doc/compute-input.md]: missing header line

File: doc/compute-job-configuration.md [hunk 2]
```diff
@@ -41,91 +41,49 @@ The main configuration files includes various global parameters used by the engi
...
-  Possible values: `DEBUG` (default), `INFO`, `WARNING`, `ERROR`, `CRITICAL`. Defines the log level that is used by the  compute engine.
-
-- `hadoop_log_root` (Required)
-
-   Hadoop clients log level and log appender. If the user wants the hadoop components to log via SYSLOG must make sure to define an appropriate appender  in hadoop log4j.properties file. The name of this appender must be added in this parameter.
-
-   For example if the available appenders in the log4j.properties file are SYSLOG and console the above line will be:
+| Name | Type | Description | Required|
+|------|------|-------------|---------|
+|`log_mode`| String | This parameter specifies the log_mode used by compute engine. Possible values: `syslog` (default), `file`, `none`.
```
Comment (dpavlos) [inline doc/compute-job-configuration.md]: by "the" compute engine

File: doc/compute-job-configuration.md [hunk 3]
```diff
@@ -162,33 +114,17 @@ This section contains various optional parameters used for correctly mapping res
...
-- `n_eg={STRING}`
-
-  Endpoint group name type used in status detailed calculations. For e.g. `n_eg=site` if site is used as a group type
-
-- `n_gg={STRING}`
 
-  Group of groups name type used in status detailed calculations
+| Name | Description | Required|
+|------|-------------|---------|
+|`e_map={fieldname1}, {fieldname2}..., {fieldnameN}`| When storing endpoint a/r results in mongodb compute engine uses the above field map to store the results using abbreviated fields. the default value is `e_map=dt,ap,p,s,n,hs,a,r,up,u,d,m,pr,ss,cs,i,sc`, where dt->date,a->availability etc... (recommended not to be changed) | `NO`|
```
Comment (dpavlos) [inline doc/compute-job-configuration.md]: "fields. the default" -> "fields. The default"

Same goes for the rest below

File: doc/compute-job-configuration.md [hunk 4]
```diff
@@ -41,91 +41,49 @@ The main configuration files includes various global parameters used by the engi
...
-
-- `hadoop_log_root` (Required)
-
-   Hadoop clients log level and log appender. If the user wants the hadoop components to log via SYSLOG must make sure to define an appropriate appender  in hadoop log4j.properties file. The name of this appender must be added in this parameter.
-
-   For example if the available appenders in the log4j.properties file are SYSLOG and console the above line will be:
+| Name | Type | Description | Required|
+|------|------|-------------|---------|
+|`log_mode`| String | This parameter specifies the log_mode used by the compute engine. Possible values: `syslog` (default), `file`, `none`.
+|`syslog`| String | The compute engine is configured to use the syslog facility, a) `file`: the compute engine can write directly to a file defined by `log_file`,  b) `none`: the compute engine does not output any logs| `YES`|
```
Comment (pkoro) [inline doc/compute-job-configuration.md]: I think this line should be part of the previous one. `log_mode` is `syslog`, `file` or `none`. It should be explained in one line.

**Pull request merged**